portal/server/controllers/dashboard/facilities/facilities-filters-query.js

```diff
@@ -40,7 +43,8 @@ const dashboardFacilitiesFiltersQuery = (filters, user) => {
       const fieldName = Object.keys(filterObj)[0];
       const filterValue = filterObj[fieldName];
 
-      const isKeywordField = fieldName === CONTENT_STRINGS.DASHBOARD_FILTERS.BESPOKE_FIELD_NAMES.KEYWORD;
+      const isKeywordField = fieldName === BESPOKE_FIELD_NAMES.KEYWORD;
+      const isStageField = fieldName === BESPOKE_FIELD_NAMES.STAGE;
```

Comment:
Once we start doing things like this, we should start thinking about a constant and a switch maybe (the constant can just be declared locally and not exported), but not for this PR

Reply:
i think i can quite easily make this a switch which would be a bit nicer

Reply:
I want to make it use `map` rather than `forEach` and `push` but had a play and was quite fiddly

Reply:
This wsa amazing good job. Thanks. A reducer would probably be better? Resolve once youve read -- just a future suggestion